tests/test_ell1h.py

```diff
@@ -143,6 +145,20 @@ def test_J0613_STIG(toasJ0613, modelJ0613_STIG):
     f.fit_toas()
 
 
+def test_SINI_error():
```

Comment:
No error is produced, and this test does not test for one. It's not very clear what is supposed to be signalled - please test on the content of the warning, both so the test is more stringent and so readers can see what is being tested.